utils/unittest/unittest.go

```diff
@@ -49,6 +49,17 @@ func (s SkipReason) String() string {
 
 func SkipUnless(t *testing.T, reason SkipReason, message string) {
 	t.Helper()
+	skippedTestList := os.Getenv("SKIPPED_TEST_LIST")
```

Comment:
can you add some explanation of what's happening here? We're checking the skipped test list to skip all the tests on that list?

Reply:
Sorry, that's outdated. I've updated the PR to use a different method for tracking skipped tests